tggs/MoDiscoIbexTGG/Gurobi_ILP.log

```diff
@@ -0,0 +1,204 @@
+
```

Comment:
Seriously!!!???  ARRRRGGGGGHHHHHHHHHHHhhhhhhhh

Reply:
War schon in der gitignore, aber die kam erst in einem späteren commit dazu